css-fonts-4/Overview.bs

```diff
@@ -6146,7 +6146,9 @@ Web Fonts shadow Installed Fonts, so if an Installed Font has the same family na
 <h3 id="preinstalled-and-user-installed-fonts">
 Preinstalled Fonts and User-Installed Fonts</h3>
 
-Users may choose to install fonts on their devices. User-Installed Fonts are installed by an explicit action by the user, such as clicking an "Install" button or copying a file into a particular directory on their device. Such fonts are User-Installed Fonts and also are Installed Fonts. Web content authors should not expect the presence of user-installed fonts, because there is no guarantee any user will have performed the action to install a specific font. User Agents may choose to ignore User-Installed Fonts for the purpose of the <a href="#font-matching-algorithm">Font Matching Algorithm</a>.
+Users may choose to install fonts on their devices. User-Installed Fonts are installed by an explicit action by the user, such as clicking an "Install" button or copying a file into a particular directory on their device. Such fonts are User-Installed Fonts and also are Installed Fonts. Web content authors should not expect the presence of user-installed fonts, because there is no guarantee any user will have performed the action to install a specific font.
+
+Note: The set of <a href="#installed-fonts">installed fonts</a> available in the <a href="#font-matching-algorithm">Font Matching Algorithm</a> is explicitly undefined. The available set of fonts <a href="https://panopticlick.eff.org">is</a> used by trackers to fingerprint users and reduce their privacy. However, some <a href="#installed-fonts">installed fonts</a>, even some <a href="#preinstalled-and-user-installed-fonts">user-installed fonts</a>, are required to make languages readable. User Agents may choose to make all installed fonts available for language support and design integrity reasons, or may choose to make some fonts unavailable for privacy reasons. In addition, User Agents may have additional facilities for fine-tuning this balance, such as interfaces to allow users to explicitly make certain fonts available or unavailable. Different User Agents, even running on the same Operating System, are expected to strike different balances here.
```

Comment:
If you want to allow the UA to exclude certain installed fonts from the list of fonts available in the Font Matching Algorithm, add an explicit normative statement that says that. Adding a statement here that the set of fonts available to the Font Matching Algorithm is undefined is disingenuous. Also, putting it in a note implies that it's a restatement of a normative definition elsewhere.

If you want to assert that such a normative definition exists because you haven't explicitly forbidden such subsetting, then it's not clear to me that the allowance is limited to installed fonts only: if you're allowing subsetting of installed by omission of forbidding it, doesn't that mean you're also allowing subsetting of web fonts, because that also is not forbidden?

Reply:
This focuses totally on UAs, with little mention of user action to allow or disallow particular fonts (per-site or globally).

Reply:
> If you want to allow the UA to exclude certain installed fonts from the list of fonts available in the Font Matching Algorithm, add an explicit normative statement that says that.

Strongly agree.